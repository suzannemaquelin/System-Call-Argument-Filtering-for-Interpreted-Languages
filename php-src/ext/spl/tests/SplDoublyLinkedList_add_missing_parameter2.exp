Warning: SplDoublyLinkedList::add() expects exactly 2 parameters, 1 given in %s on line %d
NULL