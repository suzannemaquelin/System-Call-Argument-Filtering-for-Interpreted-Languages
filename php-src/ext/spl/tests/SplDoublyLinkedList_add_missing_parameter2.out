Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Warning: SplDoublyLinkedList::add() expects exactly 2 parameters, 1 given in /root/php-src/ext/spl/tests/SplDoublyLinkedList_add_missing_parameter2.php on line 3
NULL